daemon/logger/context.go

```diff
@@ -92,7 +92,10 @@ func (ctx *Context) FullID() string {
 
 // Name returns the ContainerName without a preceding '/'.
 func (ctx *Context) Name() string {
-	return ctx.ContainerName[1:]
+	if strings.HasPrefix(ctx.ContainerName, "/") {
+		return ctx.ContainerName[1:]
+	}
+	return ctx.ContainerName
```

Comment:
Perhaps better to use `TrimPrefix`, which does exactly the same; https://golang.org/src/strings/strings.go?s=16615:16655#L605

```go
return strings.TrimPrefix(ctx.ContainerName, "/")
```